core-bundle/src/Image/ImageSizes.php

```diff
@@ -49,7 +50,7 @@ class ImageSizes
     private $predefinedSizes = [];
 
     /**
-     * @var array
+     * @var array|null
```

Comment:
This fix is wrong. The `reset()` method should use `$this->options = [];` instead, shouldn't it?

Reply:
no, the uninitialized state is `null`, the property does not have a default value

Reply:
You are right. Sorry.